File: AppAndroid/app/src/main/java/com/blog/app/domain/model/Blog.kt
package com.blog.app.domain.model

import com.blog.app.data.database.entities.BlogEntity
import com.blog.app.data.model.BlogModel

data class Blog(val id:Int,val titulo: String, val autor: String, val fecha: String, val contenido: String)


fun BlogModel.toDomain() = Blog(0,
    titulo = titulo,
    autor = autor,
    fecha = fecha,
    contenido = contenido
)

fun BlogEntity.toDomain() = Blog(id=id,
    titulo = titulo,
    autor = autor,
    fecha = fecha,
    contenido = contenido
)

File: WebAPI/blog/Model/Blog.cs
using System.ComponentModel.DataAnnotations;

public class Blog
{
    [Key]
    public int Id { get; set; }

    [Required]
    public required string Titulo { get; set; }

    [Required]
    public required string Autor { get; set; }
    
    [Required]
    public required string Contenido { get; set; }

    [Required]
    public required string Fecha { get; set; }
}